packages/venia-ui/upward.yml

```diff
@@ -100,6 +100,10 @@ veniaAppShell:
         urlResolver: veniaUrlResolver
         env: env
         mediaBackendURL: mediaBackendURL
+        site:
+          inline:
+            name:
+              inline: Venia
```

Comment:
Why are these changes needed? I'm assuming this is just changing the title to "Venia" whenever it is supplied by Upward?

Reply:
Did you not also have to change any part of the templates themselves?

Reply:
This is needed for the product template that needs `site.name`. Others don't as of now. I can create a new PR that changes other templates as well to use `site.name`.